CHANGELOG.md

```diff
@@ -1,5 +1,85 @@
+
+**Misc Changes:**
+* cilium, gops: remap to fixed port to avoid collision with nodeport range (Backport PR #14419, Upstream PR #14329, @borkmann)
+* docs: Add cgroups kernel config requirements (Backport PR #14538, Upstream PR #14517, @joestringer)
+* docs: add info about tailcalls in bpf subprograms (Backport PR #14538, Upstream PR #13888, @mfijalko)
+* docs: Clarify from/toRequires documentation with a new example (Backport PR #14308, Upstream PR #14262, @pchaigno)
+* docs: Document expected behavior for node-local DNS (Backport PR #14405, Upstream PR #14297, @aditighag)
+* docs: Fix connectivity check output (Backport PR #14308, Upstream PR #14278, @errordeveloper)
+* docs: Fix dependency conflict (Backport PR #14308, Upstream PR #14264, @joestringer)
+* docs: Fix values.yaml upgrade guide to match helm args (Backport PR #14308, Upstream PR #14237, @joestringer)
+* docs: Update linux distribution compatibility (Backport PR #14538, Upstream PR #14434, @joestringer)
+* Fix bug Cilium hangs with kvstore configured (#14629, @aanm)
+* helm: 'bpf.ctTcpMax' and 'bpf.ctAnyMax' need to be strings, not integers (Backport PR #14538, Upstream PR #14021, @mvisonneau)
+* helm: Do not deploy Hubble mTLS secrets unless Relay is enabled (Backport PR #14443, Upstream PR #14394, @gandro)
+* helm: fix TLS cert server name for cluster names containing dots (Backport PR #14538, Upstream PR #14413, @kaworu)
+* helm: fix TLS cert server name for cluster names containing dots with certgen (Backport PR #14538, Upstream PR #14416, @kaworu)
+* hubble relay: various logging improvements (Backport PR #14613, Upstream PR #14521, @kaworu)
+* jenkinsfile: Allow enabling host firewall in k8s-all CI (Backport PR #14644, Upstream PR #14524, @pchaigno)
+* microk8s: fix  add-on-command for enabling cilium (Backport PR #14405, Upstream PR #14325, @brandshaide)
+* pkg/datapath: fix arp ping handling (Backport PR #14613, Upstream PR #14501, @aanm)
+* pkg/endpoint: Readd GetRealizedPolicyRuleLabelsForKey (Backport PR #14308, Upstream PR #14257, @gandro)
+* pkg/logging: Init klog with flag set name (Backport PR #14538, Upstream PR #14346, @fafucoder)
+* pkg/node: fix concurrent access of entry node (Backport PR #14613, Upstream PR #14591, @aanm)
+* test: Add missing gomega Eventually intervals (Backport PR #14538, Upstream PR #14388, @jrajahalme)
+* Update policy-creation.rst (Backport PR #14538, Upstream PR #14241, @gecube)
+* vendor: Update vishvananda/netlink (Backport PR #14538, Upstream PR #14513, @pchaigno)
+
+**Other Changes:**
+* [v1.9] docker: bump cilium-iproute2 image (#14619, @qmonnet)
+* backport 1.9: vendor: Bump github.com/cilium/arping (#14637, @brb)
+* policy: Don't nil an empty selectors map. (#14391, @jrajahalme)
+* policy: Track selectors that contribute to MapStateEntries (#14362, @jrajahalme)
+* v1.9 backport: node: Remove check whether nextHop is in same L2 (#14453, @brb)
```

Comment:
This is the backport of https://github.com/cilium/cilium/pull/14354 which has `release-note/bug`.
@aanm Not sure if @brb could have done anything for the script to pick this up correctly?

Reply:
@pchaigno https://github.com/cilium/cilium/pull/14354 was never meged

Reply:
I've moved this to the "bug" section.